calendar_event_report/tests/test_weekly_report_wizard.py

```diff
@@ -0,0 +1,146 @@
+            'groups_id': [(6, 0, [self.RESOURCE_CALENDAR_MANAGER.id])]
+        })
+        self.employee_base = self.Employees.create({
+            'name': 'Base User',
+            'work_email': 'base@test.com',
+            'user_id': self.user_base.id,
+        })
+        self.employee_guest = self.Employees.create({
+            'name': 'Guest',
+            'work_email': 'guest@test.com',
+            'user_id': self.user_guest.id,
+        })
+        self.employee_editor = self.Employees.create({
+            'name': 'Editor',
+            'work_email': 'editor@test.com',
+            'user_id': self.user_editor.id,
+        })
+        self.employee_manager = self.Employees.create({
+            'name': 'Manager',
+            'work_email': 'manager@test.com',
+            'user_id': self.user_manager.id,
+        })
+
+    def test_010_get_current_monday(self):
+        monday = self.weekly_report_wizard_1._get_current_monday().weekday()
+        self.assertEqual(monday, 0)
+
+    def test_020_get_current_friday(self):
+        friday = self.weekly_report_wizard_1._get_current_friday().weekday()
+        self.assertEqual(friday, 4)
+
+    def test_030_default_date_start(self):
+        start_weekday = datetime.datetime.strptime(
+            self.weekly_report_wizard_1.date_start,
+            '%Y-%m-%d'
+        ).weekday()
+        self.assertEqual(start_weekday, 0)
+
+    def test_040_default_date_end(self):
+        end_weekday = datetime.datetime.strptime(
+            self.weekly_report_wizard_1.date_end,
+            '%Y-%m-%d'
+        ).weekday()
+        self.assertEqual(end_weekday, 4)
+
+    def test_050_get_report(self):
+        report = self.weekly_report_wizard_1.get_report()
+        self.assertEqual(
+            report['data']['form']['room_id'],
+            self.room_1.id
+        )
+        self.assertEqual(
+            report['data']['form']['date_start'],
+            self.weekly_report_wizard_1.date_start
+        )
+        self.assertEqual(
+            report['data']['form']['date_end'],
+            self.weekly_report_wizard_1.date_end
+        )
+        self.assertEqual(
+            report['data']['form']['recurrency'],
+            self.weekly_report_wizard_1.recurrency
+        )
+        self.assertEqual(
+            report['data']['form']['state'],
+            self.weekly_report_wizard_1.state
+        )
+
+    def has_access_to_menu(self, user_id, menu_ref):
+        user = self.env['res.users'].browse(user_id)
+        for group in user.groups_id:
+            for m in group.menu_access:
+                if m.name == self.env.ref(menu_ref).name:
+                    return True
+        return False
+
+    def test_060_base_user_cannot_get_weekly_report_menu(self):
+        self.assertFalse(self.has_access_to_menu(
+            self.user_base.id,
+            'calendar_event_report.menu_event_reports'))
+
+    def test_070_guest_user_cannot_get_weekly_report_menu(self):
```

Comment:
move tests with reference to project_resource_calendar users to project_specific